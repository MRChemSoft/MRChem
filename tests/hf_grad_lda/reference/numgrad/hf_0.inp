world_prec = 1.0e-6                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n
world_unit = angstrom

# use Psi4 naming of displacements
# displacement 1 of 5 (i.e. starting geometry) 
Molecule {
$coords
F           -1.194868474027    -0.244625419775     0.340936636348
H           -1.672505086978    -0.395082674713     1.107306807211
$end
translate = false
}

WaveFunction {
  method = LDA                      # Wave function method (HF or DFT)
}

SCF {
  guess_type = CORE_DZ             # Type of initial guess: none, mw, gto
}
