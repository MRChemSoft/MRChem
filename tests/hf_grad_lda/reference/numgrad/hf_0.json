{
  "input": {
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "atom": "h",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf_0.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": false,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "restricted": true,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 5,
        "localize": false,
        "max_iter": 100,
        "method": "DFT (SVWN5)",
        "orbital_thrs": 9.999999999999999e-06,
        "rotation": 0,
        "start_prec": 1e-06
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.3034482007428676,
        -0.47659952984111875,
        0.7172401613655944
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.7053216686702017,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.36799094936152343,
            -0.11591865961557168,
            0.5904440547898266
          ],
          "vector_el": [
            23.11435314476598,
            4.791154810685754,
            -7.300554363724316
          ],
          "vector_nuc": [
            -23.4823440941275,
            -4.9070734703013255,
            7.890998418514143
          ]
        }
      },
      "geometric_derivative": {
        "electronic": [
          0.0,
          0.0,
          0.0
        ],
        "electronic_norm": 0.0,
        "nuclear": [
          0.0,
          0.0,
          0.0
        ],
        "nuclear_norm": 0.0,
        "total": [
          0.0,
          0.0,
          0.0
        ],
        "total_norm": 0.0
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -24.0876156669762,
          -1.097571162748223,
          -0.49930447563154773,
          -0.36104647800353806,
          -0.36104645038606625
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -52.813168467491145
      },
      "scf_energy": {
        "E_ee": 55.468202385469645,
        "E_eext": 0.0,
        "E_el": -105.0542104350533,
        "E_en": -249.80784546012126,
        "E_kin": 99.425527520518,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -99.85189829403437,
        "E_x": 0.0,
        "E_xc": -10.140094880919726
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 20,
      "routine": "mrchem.x",
      "total_cores": 20,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.59019563648894,
        "E_eext": 0.0,
        "E_el": -83.24217250137998,
        "E_en": -329.27316650860746,
        "E_kin": 166.7481855430811,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -78.03986036036105,
        "E_x": 0.0,
        "E_xc": -15.307387172342564
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 58.078696276119004,
              "E_eext": 0.0,
              "E_el": -104.43270330036813,
              "E_en": -247.2382697453263,
              "E_kin": 95.02519580776413,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.2303911593492,
              "E_x": 0.0,
              "E_xc": -10.298325638924968
            },
            "energy_total": -99.2303911593492,
            "energy_update": 21.19053079898815,
            "mo_residual": 1.1964120554181177,
            "wall_time": 25.393666475
          },
          {
            "energy_terms": {
              "E_ee": 50.81552401267854,
              "E_eext": 0.0,
              "E_el": -104.74519249907712,
              "E_en": -239.43952636744694,
              "E_kin": 93.38727256947617,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.54288035805818,
              "E_x": 0.0,
              "E_xc": -9.508462713784892
            },
            "energy_total": -99.54288035805818,
            "energy_update": 0.31248919870898817,
            "mo_residual": 0.500441789749252,
            "wall_time": 24.026504623
          },
          {
            "energy_terms": {
              "E_ee": 57.26209538656039,
              "E_eext": 0.0,
              "E_el": -104.99836778339815,
              "E_en": -251.9890468523234,
              "E_kin": 100.06520612638607,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.79605564237922,
              "E_x": 0.0,
              "E_xc": -10.3366224440212
            },
            "energy_total": -99.79605564237922,
            "energy_update": 0.2531752843210313,
            "mo_residual": 0.36337284395876884,
            "wall_time": 21.566139632
          },
          {
            "energy_terms": {
              "E_ee": 54.94422558168105,
              "E_eext": 0.0,
              "E_el": -105.03895104691297,
              "E_en": -248.3759802962496,
              "E_kin": 98.4506259623515,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.83663890589403,
              "E_x": 0.0,
              "E_xc": -10.057822294695919
            },
            "energy_total": -99.83663890589403,
            "energy_update": 0.04058326351481867,
            "mo_residual": 0.12738899175881727,
            "wall_time": 21.869430611
          },
          {
            "energy_terms": {
              "E_ee": 55.77172145317749,
              "E_eext": 0.0,
              "E_el": -105.05201428939276,
              "E_en": -250.39823204110778,
              "E_kin": 99.75122275417678,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.84970214837382,
              "E_x": 0.0,
              "E_xc": -10.176726455639253
            },
            "energy_total": -99.84970214837382,
            "energy_update": 0.013063242479788073,
            "mo_residual": 0.07841563750495095,
            "wall_time": 23.753972635
          },
          {
            "energy_terms": {
              "E_ee": 55.440838174290406,
              "E_eext": 0.0,
              "E_el": -105.05391876811478,
              "E_en": -249.7783970492455,
              "E_kin": 99.41967790243555,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85160662709585,
              "E_x": 0.0,
              "E_xc": -10.136037795595222
            },
            "energy_total": -99.85160662709585,
            "energy_update": 0.0019044787220252601,
            "mo_residual": 0.022045164967886388,
            "wall_time": 26.889036176
          },
          {
            "energy_terms": {
              "E_ee": 55.465890941634285,
              "E_eext": 0.0,
              "E_el": -105.05419319004253,
              "E_en": -249.81661670347256,
              "E_kin": 99.43657130704317,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85188104902359,
              "E_x": 0.0,
              "E_xc": -10.140038735247398
            },
            "energy_total": -99.85188104902359,
            "energy_update": 0.00027442192774174146,
            "mo_residual": 0.009630862027013659,
            "wall_time": 24.189969564
          },
          {
            "energy_terms": {
              "E_ee": 55.469991683088665,
              "E_eext": 0.0,
              "E_el": -105.05420746625795,
              "E_en": -249.81360814374796,
              "E_kin": 99.42977052524895,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85189532523901,
              "E_x": 0.0,
              "E_xc": -10.140361530847615
            },
            "energy_total": -99.85189532523901,
            "energy_update": 1.4276215424047223e-05,
            "mo_residual": 0.0011833158758959034,
            "wall_time": 25.33761192
          },
          {
            "energy_terms": {
              "E_ee": 55.46832912198338,
              "E_eext": 0.0,
              "E_el": -105.05421018388145,
              "E_en": -249.80570000118414,
              "E_kin": 99.42323529797203,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85189804286252,
              "E_x": 0.0,
              "E_xc": -10.140074602652703
            },
            "energy_total": -99.85189804286252,
            "energy_update": 2.7176235022352557e-06,
            "mo_residual": 0.0007093804771258717,
            "wall_time": 24.565811638
          },
          {
            "energy_terms": {
              "E_ee": 55.46799887581954,
              "E_eext": 0.0,
              "E_el": -105.05421036922955,
              "E_en": -249.80613752912703,
              "E_kin": 99.42397702771171,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85189822821062,
              "E_x": 0.0,
              "E_xc": -10.140048743633786
            },
            "energy_total": -99.85189822821062,
            "energy_update": 1.8534809953507647e-07,
            "mo_residual": 0.00017883767663488584,
            "wall_time": 24.566876788
          },
          {
            "energy_terms": {
              "E_ee": 55.46825832802611,
              "E_eext": 0.0,
              "E_el": -105.0542104301127,
              "E_en": -249.80816151206483,
              "E_kin": 99.42580126787475,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85189828909377,
              "E_x": 0.0,
              "E_xc": -10.140108513948746
            },
            "energy_total": -99.85189828909377,
            "energy_update": 6.08831527415532e-08,
            "mo_residual": 9.637380210545506e-05,
            "wall_time": 25.331927645
          },
          {
            "energy_terms": {
              "E_ee": 55.46817562222418,
              "E_eext": 0.0,
              "E_el": -105.05421043381082,
              "E_en": -249.80803666351989,
              "E_kin": 99.42574805950952,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85189829279189,
              "E_x": 0.0,
              "E_xc": -10.140097452024632
            },
            "energy_total": -99.85189829279189,
            "energy_update": 3.698119144246448e-09,
            "mo_residual": 2.087715961162531e-05,
            "wall_time": 26.912969157
          },
          {
            "energy_terms": {
              "E_ee": 55.46824672249988,
              "E_eext": 0.0,
              "E_el": -105.05421043477646,
              "E_en": -249.80803318783381,
              "E_kin": 99.42567900742472,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85189829375753,
              "E_x": 0.0,
              "E_xc": -10.140102976867254
            },
            "energy_total": -99.85189829375753,
            "energy_update": 9.656417887526914e-10,
            "mo_residual": 1.7610892000193722e-05,
            "wall_time": 24.706838464
          },
          {
            "energy_terms": {
              "E_ee": 55.468202385469645,
              "E_eext": 0.0,
              "E_el": -105.0542104350533,
              "E_en": -249.80784546012126,
              "E_kin": 99.425527520518,
              "E_next": 0.0,
              "E_nn": 5.2023121410189415,
              "E_nuc": 5.2023121410189415,
              "E_tot": -99.85189829403437,
              "E_x": 0.0,
              "E_xc": -10.140094880919726
            },
            "energy_total": -99.85189829403437,
            "energy_update": 2.7684166070685023e-10,
            "mo_residual": 8.597682323662736e-06,
            "wall_time": 25.172595396
          }
        ],
        "wall_time": 344.287988967
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
