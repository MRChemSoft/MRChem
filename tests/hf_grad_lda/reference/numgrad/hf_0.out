                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         mrchem-gradient                          ***
***         Git commit hash    c6b193dd4223e5895077                     ***
***         Git commit author  Roberto Di Remigio                       ***
***         Git commit date    Fri Sep 25 22:32:15 2020 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                              20
 Total cores             :                                              20
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.3.1
 Git branch            : mrchem-gradient
 Git commit hash       : b79bdf06c3c82a53edb6
 Git commit author     : Roberto Di Remigio
 Git commit date       : Tue Sep 15 18:31:50 2020 +0200

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : MPI/OpenMP

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999932277820e-01
    1    2    p          :                              9.999512243981e-01
    2    2    p          :                              9.999999998838e-01
    3    2    p          :                              9.999999964633e-01
    4    2    p          :                              9.999999871779e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                166.748185543081
 E-N energy              :            (au)               -329.273166508607
 Coulomb energy          :            (au)                 94.590195636489
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.307387172343
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.202312141019
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -83.242172501380
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.803986036036e+01
                         :      (kcal/mol)             -4.897075133556e+04
                         :        (kJ/mol)             -2.048936237210e+05
                         :            (eV)             -2.123572689514e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -14.606577995652
    1    2    p          :            (au)                  3.225671980995
    2    2    p          :            (au)                  4.747947394922
    3    2    p          :            (au)                  4.773693296088
    4    2    p          :            (au)                  5.078812324610
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  6.439094001926
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (SVWN5)
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          2.236068e+00         -78.039860360361         -7.803986e+01
    1          1.196412e+00         -99.230391159349         -2.119053e+01
    2          5.004418e-01         -99.542880358058         -3.124892e-01
    3          3.633728e-01         -99.796055642379         -2.531753e-01
    4          1.273890e-01         -99.836638905894         -4.058326e-02
    5          7.841564e-02         -99.849702148374         -1.306324e-02
    6          2.204516e-02         -99.851606627096         -1.904479e-03
    7          9.630862e-03         -99.851881049024         -2.744219e-04
    8          1.183316e-03         -99.851895325239         -1.427622e-05
    9          7.093805e-04         -99.851898042863         -2.717624e-06
   10          1.788377e-04         -99.851898228211         -1.853481e-07
   11          9.637380e-05         -99.851898289094         -6.088315e-08
   12          2.087716e-05         -99.851898292792         -3.698119e-09
   13          1.761089e-05         -99.851898293758         -9.656418e-10
   14          8.597682e-06         -99.851898294034         -2.768417e-10
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 99.425527520518
 E-N energy              :            (au)               -249.807845460121
 Coulomb energy          :            (au)                 55.468202385470
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -10.140094880920
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.202312141019
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -105.054210435053
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.985189829403e+01
                         :      (kcal/mol)             -6.265801167713e+04
                         :        (kJ/mol)             -2.621611210273e+05
                         :            (eV)             -2.717108452452e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -24.087615666976
    1    2    p          :            (au)                 -1.097571162748
    2    2    p          :            (au)                 -0.499304475632
    3    2    p          :            (au)                 -0.361046478004
    4    2    p          :            (au)                 -0.361046450386
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -52.813168467491
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       23.114353        4.791155       -7.300554
 Magnitude               :            (au)                       24.708836
                         :         (Debye)                       62.803591
---------------------------------------------------------------------------
 Nuclear vector          :      -23.482344       -4.907073        7.890998
 Magnitude               :            (au)                       25.254063
                         :         (Debye)                       64.189420
---------------------------------------------------------------------------
 Total vector            :       -0.367991       -0.115919        0.590444
 Magnitude               :            (au)                        0.705322
                         :         (Debye)                        1.792749
===========================================================================


===========================================================================
                           Geometric Derivative
---------------------------------------------------------------------------
 Total                   :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
---------------------------------------------------------------------------
 Nuclear                 :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
---------------------------------------------------------------------------
 Electronic              :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  5m 51s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
